.gitignore

```diff
@@ -4,3 +4,10 @@
 /.cabal-sandbox
 /cabal.sandbox.config
 result*
+
+# AFL stub intermediates
```

Comment:
Probably should just add this as another test-suite component in the cabal file, which keeps it all under dist/dist-newstyle. I wouldn't object to adding `*.o` and `*.hi` rules though.